server/test/unit/server/tasks/test_repository.py

```diff
@@ -176,8 +176,8 @@ def test_pass_through_to_manager(self, mock_queue_publish):
 class TestRepositorySync(unittest.TestCase):
     @patch('pulp.server.managers.repo.sync.RepoSyncManager.queue_sync_with_auto_publish')
     def test_pass_through_to_manager(self, mock_queue_sync):
-        result = repository.sync_with_auto_publish('foo', {})
+        result = repository.sync_with_auto_publish('foo', {}, None)
         # make sure the args get passed through
-        mock_queue_sync.assert_called_once_with('foo', {})
+        mock_queue_sync.assert_called_once_with('foo', {}, None)
```

Comment:
Consider using something other than None for the arg you add. I suggest a string.